modules/open62541/1.4.1/patches/add_build_file.patch

```diff
@@ -0,0 +1,17 @@
+diff --git /dev/null BUILD.bazel
+new file mode 100644
+index 00000000..f421d34f
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,10 @@
++cc_library(
++    name = "open62541",
++    srcs = ["lib/libopen62541.a"],
```

Comment:
Are you sure `lib/libopen62541.a` is correct? I don't see this file in the source archive.

Reply:
let me test this locally again